average.js

```diff
@@ -1,6 +1,12 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+    
+  for(var i=0;i<numbers.length;i++){
+      if(isNaN(numbers[i])){
+          numbers.splice(i,1);
+      }
+  }
+return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
```

Comment:
See if you can make the code easier to read... `numbers` starts as the input of the sensor and ends as the sanitized list. This could make the function harder to maintain as it grows. What do you think?